### DISCUSSION TITLE
為什麼網頁會發生一直跳動的問題？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#266] [Q&A] [answered]
[老師的github統計點數網站https://pypal.ncuedu.tw/about](https://pypal.ncuedu.tw/about)老師的這個網站會一直跳動，不知為什麼會這樣。這個問題或許我們以後自己做網頁也會遇到。

**Comment by chia-kai-chang** [comment, marked as answer]
哈哈，我也不知道，應該javascript程式互鎖了
